astra/1.7/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.astralinux.ru/library/alse:1.7.3uu2-mg9.2.0
```

Comment:
1. The latest version is 1.7.4.
2. It can be `registry.astralinux.ru/library/alse:1.7.4`

Reply:
fixed